lms/templates/edxnotes/note-item.underscore

```diff
@@ -37,7 +37,7 @@
 
         <% if (tags.length > 0) { %>
             <h3 class="reference-title"><%- gettext("Tags:") %></h3>
-            <span class="reference-meta reference-tags"><%- tags.join(", ") %></span>
+            <span class="reference-meta reference-tags"><%= tags.join(", ") %></span>
```

Comment:
Yes, we need the HTML because the highlighted sections are represented by injected span tags.